packages/connector-oauth2/docs/config-template.json

```diff
@@ -0,0 +1,10 @@
+{
+  "oauthGrantType": "AuthorizationCode",
+  "authorizationEndpoint": "<authorization-endpoint>",
+  "tokenEndpoint": "<token-endpoint>",
+  "userInfoEndpoint": "<user-info-endpoint>",
+  "redirectUri": "<redirect-uri>",
+  "clientId": "<client-id>",
+  "clientSecret": "<client-secret>",
+  "scope": "<scope>"
```

Comment:
Do we need to add "profileMapping" example?

Reply:
Added. Will elaborate on the configuration of different flows in README.md.